frontend/src/api/upload-api.js

```diff
@@ -0,0 +1,27 @@
+import { storage } from "../config/firebase";
+
+/* function will store images in images directory  */
+/* arguments : file to upload */
+export const uploadImage = (data) => {
+  return new Promise((resolve, reject) => {
+    // const userId = localStorage.getItem("id");
+    const userId = "123";
```

Comment:
userId is not constant.

Reply:
sorry , i used this for testing , will fix that ..